pkg/operator/ceph/object/user.go

```diff
@@ -54,7 +58,20 @@ func decodeUser(data string) (*ObjectUser, int, error) {
 		return nil, RGWErrorParse, errors.Wrapf(err, "failed to unmarshal json. %s", data)
 	}
 
-	rookUser := ObjectUser{UserID: user.ID, DisplayName: &user.DisplayName, Email: &user.Email}
+	rookUser := ObjectUser{UserID: user.ID, DisplayName: &user.DisplayName, Email: &user.Email, MaxBuckets: *user.MaxBuckets}
```

Comment:
`user.MaxBuckets` isn't always set, right? Seems like it could cause a null ref.